logic/pricesuggest/algorithm.py

```diff
@@ -0,0 +1,43 @@
+class pricesuggest:
+    """
+
+    ok so over here im just going to suggest the variables and try to classify them
+
+    Primary variable(variabels which affect each other):
+        -square feet
+        -amount of rooms
+        -amount of bathrooms
+
+
+    Secondary Variables(variables whose effect is constent):
+        -Pool
+        -Location
+        -age(not sure about this one)
```

Comment:
Hmm age would be best to leave out- I know some houses that are expensive because they are brand new, and some that are expensive because they are antique. Maybe y = -x^2 or something? Would be pretty tough to get right though.